### PR TITLE
Popup icon is gray when disabled

### PR DESCRIPTION
**Pull request by Atli-A**
Fixing issue #2

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -52,22 +71,22 @@ function request_send_status_updates() {
     }, tabs => {
         if (tabs.length > 0) {
             active_tab_id = tabs[0].id;
-            chrome.tabs.sendMessage(active_tab_id, {what: "send_status_updates"}, checkLastError);
```
Comment (wmww) [inline src/popup.js]: Can you remove this and other formatting changes? I have no strong feelings about formatting, except that unrelated code shouldn't be changed in a feature PR.

**Review by wmww — COMMENTED** [review]
Thanks! Looks good at first glance. Just a heads up I don't think this is important enough to make a release for, so once it merges it might be a while before it makes it into the stores.

File: src/popup.js (hunk 1 above)
Comment (Atli-A) [reply src/popup.js]: Yes, I will fix that later today.

edit: fixed it.

**Comment by wmww** [comment]
Sorry for the delay, been busy.

There seems to be a bit of an issue with startup. To reproduce:
1. Load the extension and turn it off
2. in about:debugging#/runtime/this-firefox (where you load the extension), click reload
Expected: It will be off and icon will be gray
Actual: It's off but icon doesn't turn gray until you open up the popup

The issue seems to be that the popup script isn't loaded until the popup is opened, but moving the icon setting to the main script doesn't work. Other extensions change icon dynamically so there must be a way.

**Comment by wmww** [comment]
Same behavior with Chrome, except it's more obvious how to test it since Chrome doesn't unload local extensions on exit. Simply turn it off, close chrome and re-open chrome. It stays off, but doesn't gray out the icon until you open the popup.

**Comment by wmww** [comment]
Ah! Dug through the source code of a couple other extensions that do it and it looks like we need a background script

**Comment by wmww** [comment]
I seem to have screwed up this branch. Git is being weird. I pushed my additions to a new PR #21. Would appropriate someone else to test it before I merge.

**Comment by Atli-A** [comment]
That's interesting I was unable to reproduce on chrome but I did run into the same issue on firefox. I guess I just didn't test for that. Thanks for the fix!
